signalfx/resource_signalfx_aws_integration.go

```diff
@@ -749,12 +757,15 @@ func waitForIntegrationSpecificSyncStateToSettle(d *schema.ResourceData, syncSta
 	return int.(*integration.AwsCloudWatchIntegration), nil
 }
 
-func integrationAWSDelete(d *schema.ResourceData, meta interface{}) error {
-	// Do nothing, let the aws_(external|token)_integration do the deletion
+func integrationAWSDelete(d *schema.ResourceData, meta any) error {
+	return DoIntegrationAWSDelete(d, meta)
+}
+
+func noop(_ *schema.ResourceData, _ any) error {
 	return nil
 }
 
-func validateFilterAction(v interface{}, k string) (we []string, errors []error) {
+func validateFilterAction(v any, _ string) (we []string, errors []error) {
```

Comment:
can we remove the second param?

Reply:
I don't think so as this function has to match the TF's `SchemaValidateFunc` signature. 

Without the second param there's this error:
`Cannot use 'validateFilterAction' (type func(v any) (we []string, errors []error)) as the type SchemaValidateFunc`

Reply:
we can consider replacing it at some point

https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#SchemaValidateFunc